src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -25,18 +25,32 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
   }
 
   implicit val eth_submitHashrate = new JsonDecoder[SubmitHashRateRequest] with JsonEncoder[SubmitHashRateResponse] {
+    val lengthWithPrefix = 66
+
     override def decodeJson(params: Option[JsonAST.JArray]): Either[JsonRpcError, SubmitHashRateRequest] = params match {
-      case Some(JArray(hashRate :: JString(id) :: Nil)) =>
-        extractQuantity(hashRate)
-          .flatMap(h => extractBytes(id).map(i => (h, i)))
-          .map { case (h, i) => SubmitHashRateRequest(h, i) }
+      case Some(JArray((hashRate: JString) :: (id: JString) :: Nil))
```

Comment:
`case Some(JArray(hashRate :: JString(id) :: Nil))`

`hashRate` - because we want to accept JSON ints as input.
`JString(id)` - because it's just nicer :)